src/apis/price-updater.ts

```diff
@@ -24,14 +47,37 @@ const mockedFiatExchangeRates = {
   GBP: 0.716945,
 };
 
+const parsedReactAppPriceUpdaterApiUrl = z
+  .string()
+  .safeParse(process.env.REACT_APP_PRICE_UPDATER_API_URL);
```

Comment:
I think that we need to find a better way to parse the entire `env` object, if not we will need to do this every time we want to access them.

Reply:
Worth considering, but for another slice. Some drawbacks of this approach may be:

 * A single decoding failure will make all the config decoding fail.
 * We will need to decode the config before the app initializes and pass the whole object as prop or context.

I don't see the current approach as a big problem, since all the parses will be under persistence. But let's give it a thought!

Reply:
Yep, I said it just to keep it in mind for the next slices! Not to be covered now as we need to think if there are a better approach or not as you said!